daemon/mgr/cri_streaming.go

```diff
@@ -0,0 +1,50 @@
+package mgr
+
+import (
+	"fmt"
+	"io"
+	"net"
+
+	k8snet "k8s.io/apimachinery/pkg/util/net"
```

Comment:
Can we  only import "k8s.io/apimachinery/pkg/util/net" , not the whole apimachinery pkg?